bridge/bridge.go

```diff
@@ -0,0 +1,14 @@
+package bridge
+
+import (
+	"github.com/giantswarm/flanneltpr/docker"
+)
+
+// Bridge holds information used for the execution of the Docker entrypoint of
+// https://github.com/giantswarm/k8s-network-bridge.
+type Bridge struct {
+	// Config contains network bridge configuration.
+	Config Config `json:"config" yaml:"config"`
+	// Docker describes the docker image running k8s-network-bridge.
+	Docker docker.Docker `json:"docker" yaml:"docker"`
```

Comment:
Because of consistency. Now I need to think and take time to understand and discover. This just adds up and does not make any sense to me to do differently everywhere. Such conventions just make our lives way more easy in the long run. And I am willing to accept a little bit more initial overhead to create another directory.